[2022-07-04 09:50:01,180] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: example_bash_operator.also_run_this manual__2022-07-04T09:49:59.473162+00:00 [queued]>
[2022-07-04 09:50:01,202] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: example_bash_operator.also_run_this manual__2022-07-04T09:49:59.473162+00:00 [queued]>
[2022-07-04 09:50:01,203] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-07-04 09:50:01,204] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-07-04 09:50:01,205] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-07-04 09:50:01,231] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): also_run_this> on 2022-07-04 09:49:59.473162+00:00
[2022-07-04 09:50:01,235] {standard_task_runner.py:52} INFO - Started process 396 to run task
[2022-07-04 09:50:01,252] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'example_bash_operator', 'also_run_this', 'manual__2022-07-04T09:49:59.473162+00:00', '--job-id', '11', '--raw', '--subdir', '/home/***/.local/lib/python3.7/site-packages/***/example_dags/example_bash_operator.py', '--cfg-path', '/tmp/tmpwoy4m3ke', '--error-file', '/tmp/tmp9s449gbl']
[2022-07-04 09:50:01,253] {standard_task_runner.py:80} INFO - Job 11: Subtask also_run_this
[2022-07-04 09:50:01,408] {task_command.py:370} INFO - Running <TaskInstance: example_bash_operator.also_run_this manual__2022-07-04T09:49:59.473162+00:00 [running]> on host b631230cc7b5
[2022-07-04 09:50:01,791] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=example_bash_operator
AIRFLOW_CTX_TASK_ID=also_run_this
AIRFLOW_CTX_EXECUTION_DATE=2022-07-04T09:49:59.473162+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-07-04T09:49:59.473162+00:00
[2022-07-04 09:50:01,792] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-07-04 09:50:01,792] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo "run_id=manual__2022-07-04T09:49:59.473162+00:00 | dag_run=<DagRun example_bash_operator @ 2022-07-04 09:49:59.473162+00:00: manual__2022-07-04T09:49:59.473162+00:00, externally triggered: True>"']
[2022-07-04 09:50:01,825] {subprocess.py:85} INFO - Output:
[2022-07-04 09:50:01,839] {subprocess.py:92} INFO - run_id=manual__2022-07-04T09:49:59.473162+00:00 | dag_run=<DagRun example_bash_operator @ 2022-07-04 09:49:59.473162+00:00: manual__2022-07-04T09:49:59.473162+00:00, externally triggered: True>
[2022-07-04 09:50:01,840] {subprocess.py:96} INFO - Command exited with return code 0
[2022-07-04 09:50:01,919] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=example_bash_operator, task_id=also_run_this, execution_date=20220704T094959, start_date=20220704T095001, end_date=20220704T095001
[2022-07-04 09:50:01,994] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-04 09:50:02,070] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
